src/Tagged.elm

```diff
@@ -39,11 +39,14 @@ map f (Tagged x) =
 {-| Useful for composing functions together in a pipeline:
 
     foo =
-        Array.set |> map index |> andMap value |> andMap arr
+        tagged Array.set
+            |> andMap index
+            |> andMap value
+            |> andMap arr
 
 -}
-andMap : Tagged tag (oldValue -> newValue) -> Tagged tag oldValue -> Tagged tag newValue
-andMap (Tagged f) (Tagged x) =
+andMap : Tagged tag oldValue -> Tagged tag (oldValue -> newValue) -> Tagged tag newValue
```

Comment:
You don't have to worry about it in this PR, but I'm going to add back the function with the arguments the other way. They're both useful functions.